ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/privacy/PrivateTransactionResult.java

```diff
@@ -53,6 +55,36 @@ public PrivateTransactionResult(final PrivateTransaction tx) {
     this.restriction = new String(tx.getRestriction().getBytes().toArrayUnsafe(), UTF_8);
   }
 
+  @JsonCreator
+  public PrivateTransactionResult(
```

Comment:
Does this need to change? Seems odd that we would be modifying classes in the API module for a AT unless there is a bug somewhere.

Reply:
Good point.  Reverted and added a new response class in the AT package.